tests/kafkatest/services/mirror_maker.py

```diff
@@ -133,12 +145,19 @@ def start_node(self, node):
         node.account.ssh("mkdir -p %s" % MirrorMaker.LOG_DIR, allow_fail=False)
 
         # Create, upload one consumer config file for source cluster
-        consumer_props = self.render('consumer.properties', zookeeper_connect=self.source.zk.connect_setting())
+        if self.new_consumer:
+            consumer_props = self.render("new_consumer.properties", bootstrap_servers=self.source.bootstrap_servers())
+        else:
+            consumer_props = self.render("consumer.properties", zookeeper_connect=self.source.zk.connect_setting())
+            consumer_props += "\noffsets.storage=%s\n" % self.offsets_storage
+
         node.account.create_file(MirrorMaker.CONSUMER_CONFIG, consumer_props)
+        self.logger.info("Mirromaker consumer props:\n" + consumer_props)
```

Comment:
Haha, yes, and I find that behavior infuriating. It's actually really annoying in Copycat where using AbstractConfig results in dumping configs to the console every time connectors or tasks are started....

Anyway, consistency is fine for now. My experience so far going through logs is that if I want an overview of what happened in the test via the INFO log, I'm scrolling a bunch just to get through config data.